src/platform/packages/shared/kbn-lens-embeddable-utils/config_builder/columns/counter_rate.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { CounterRateIndexPatternColumn, GenericIndexPatternColumn } from '@kbn/lens-plugin/public';
+import type { LensApiCounterRateOperation, LensApiFieldMetricOperations } from '../schema/metric_ops';
+import { getMetricColumnReverse, getMetricColumn } from './metric';
+
+export const getCounterRateColumn = (options: LensApiCounterRateOperation): [CounterRateIndexPatternColumn, GenericIndexPatternColumn] => {
+  return [{
```

Comment:
reference type of operations return two columns, thats why we also need to pass all columns to reverse function so it can find the referenced one.